pkg/watcher/rated.go

```diff
@@ -96,6 +102,32 @@ func getValidationStatistics(cfg *core.Config, key string) (*getValidatorEffecti
 
 			return nil, fmt.Errorf("Validator not found")
 
+		} else if res.StatusCode == 401 {
+			body, err := io.ReadAll(res.Body)
+			if err != nil {
```

Comment:
hhm not sure you need to this or the next check. You can simply log the 401 and return